spec/build_spec.rb

```diff
@@ -252,24 +252,14 @@ def pull_request?
 end
 
 describe 'Build Testing' do
-  context 'when calling query_branches' do
-    it 'should include branches that are valid and new' do
+  context 'when calling query_for_new_builds' do
```

Comment:
Lots of testing updates to account for the removed code, changed function signatures, and modified behavior.